### ISSUE TITLE
false positive on variable named temperature and Regex bug

### ISSUE THREAD

**Issue by rkam** [erica/testlint#4]
Firstly, thanks for this. It's very helpful while I try to unlearn ingrained habits when writing swift!
## 

I fixed the false positive by including word boundaries around the relevant triggers as follows:

```
        if ([RegexHelper testString:@"\\Btemp\\B" inString:line] ||
            [RegexHelper testString:@"\\Btmp\\B" inString:line] ||
            [RegexHelper testString:@"\\Bresults\\B" inString:line] ||
            [RegexHelper testString:@"\\BreturnValue\\B" inString:line])
        {
```

I'm not entirely sure about them around other than _temp_, since your intent may be to also prevent things like _tmpfoo_.  - Speaking of which, you may want to add _foo_, _bar_, _foobar_ to the list :)
## 

Also, in your original code, you don't ignore case, so, e.g. _Temp_ passes.
- Actually, after looking at the code in _RegexHelper.m_, I saw that you have the option in the wrong method. (I moved it and it worked)

(Normally, I'd submit a change, but I have several other checks disabled and so have other changes to your code. Also, the above should be clear and trivial enough that it didn't seem to be necessary either).
